test/images/busybox/BASEIMAGE

```diff
@@ -3,5 +3,5 @@ linux/arm=arm32v6/busybox:1.29
 linux/arm64=arm64v8/busybox:1.29
 linux/ppc64le=ppc64le/busybox:1.29
 linux/s390x=s390x/busybox:1.29
-windows/amd64/1809=mcr.microsoft.com/windows/nanoserver:1809
+windows/amd64/ltsc2019=mcr.microsoft.com/windows/nanoserver:ltsc2019
```

Comment:
If we're not replacing 1809 with ltsc2019, then both entries should exist. Same below.

Reply:
Updated.